front/app/containers/ProjectsShowPage/shared/poll/PollForm.tsx

```diff
@@ -122,6 +177,13 @@ const PollForm = ({ questions, id, type, disabled, projectId }: Props) => {
 
     return (
       <>
+        {authUser &&
+          !isNilOrError(disabledMessage) &&
+          actionDisabledAndNotFixable && (
+            <Box mb="16px">
+              <Warning>{formatMessage(disabledMessage)}</Warning>
+            </Box>
```

Comment:
Shows a visible warning message if the issue isn't fixable, otherwise they would only see this on the submit button hover. Thought this was more clear for users.